pkg/model/volumes.go

```diff
@@ -183,7 +210,7 @@ func (dev *Dev) PersistentVolumeStorageClass() string {
 
 func (dev *Dev) AreDefaultPersistentVolumeValues() bool {
```

Comment:
@jLopezbarb What is this function used for? Might it have some impact on users that deployed a dev volume with previous default value and they start using the new defaults?

Reply:
It's only used in https://github.com/okteto/okteto/blob/bd289c9ee57dd1a226ed9caefd58bdb07afab559/pkg/model/serializer.go#L1049 to check for marshaling of the dev struct. It's used to check if the values should be written to the marshal or skipped since they are the defaults we had set. I think it's ok to write the values while marshaling in this "migration" since the defaults have changed